mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/AddressBookScreenV2.tsx

```diff
@@ -0,0 +1,458 @@
+  const filterAddress = useCallback(debounce((searchString: string): void => {
+    setFilteredAddressBook(sortByFavourite(addressBook).filter(address =>
+      address.label.toLowerCase().includes(searchString.trim().toLowerCase()) ||
+      address.address.includes(searchString.trim().toLowerCase())
+    ))
+    setFilteredWalletAddress(sortByFavourite(walletAddress).filter(address =>
+      address.label.toLowerCase().includes(searchString.trim().toLowerCase()) ||
+      address.address.includes(searchString.trim().toLowerCase())
+    ))
+  }, 200), [addressBook, walletAddress, activeButtonGroup])
+
+  // Favourite
+  const onFavouriteAddress = async (localAddress: LocalAddress): Promise<void> => {
+    const labeledAddress = {
+      [localAddress.address]: {
+        ...localAddress,
+        isFavourite: typeof localAddress.isFavourite === 'boolean' ? !localAddress.isFavourite : true
+      }
+    }
+    dispatch(userPreferences.actions.addToAddressBook(labeledAddress))
+  }
+
+  const sortByFavourite = (localAddresses: LocalAddress[]): LocalAddress[] => {
+    return [...localAddresses].sort((curr, next) => {
+      if (curr.isFavourite === true) {
+        return -1
+      }
+      if (next.isFavourite === true) {
+        return 1
+      }
+      return 0
+    })
+  }
+
+  useEffect(() => {
+    // sync all store changes to local storage
+    const updateLocalStorage = async (): Promise<void> => {
+      await dispatch(setUserPreferences({ network, preferences: userPreferencesFromStore }))
+    }
+    updateLocalStorage().catch(Logging.error)
+  }, [userPreferencesFromStore])
+
+  useEffect(() => {
+    // update on search, on tab change
+    if (searchString.trim().length !== 0) {
+      filterAddress(searchString)
+      return
+    }
+
+    activeButtonGroup === ButtonGroupTabKey.Whitelisted
+    ? setFilteredAddressBook(sortByFavourite(addressBook))
+    : setFilteredWalletAddress(sortByFavourite(walletAddress))
+  }, [addressBook, walletAddress, searchString, activeButtonGroup])
+
+  useEffect(() => {
+    navigation.setOptions({
+      headerStyle: [...(headerStyle as Array<StyleProp<ViewStyle>>), tailwind('rounded-b-none border-b-0'), { shadowOpacity: 0 }]
+    })
+  }, [])
+
+  const AddressListItem = useCallback(({
+    item,
+    index,
+    testIDSuffix
+  }: { item: LocalAddress, index: number, testIDSuffix: string }): JSX.Element => {
+    return (
+      <ThemedViewV2
+        key={item.address}
+        light={tailwind('bg-mono-light-v2-00')}
+        dark={tailwind('bg-mono-dark-v2-00')}
+        style={tailwind('py-4.5 pl-5 pr-4 mb-2 rounded-lg-v2')}
+        testID={`address_row_${index}_${testIDSuffix}`}
+      >
+        <View style={tailwind('flex flex-row items-center flex-grow', { 'flex-auto': Platform.OS === 'web' })}>
+          {item.isMine
+            ? (
+              <View style={tailwind('mr-3')}>
+                <RandomAvatar name={item.address} size={36} />
```

Comment:
let change the `<Avatar/>` to use `variant='sunset'` and change the color palette